src/app/backend/errors/handler.go

```diff
@@ -18,6 +18,7 @@ import (
 	"log"
 	"net/http"
 
+	"github.com/emicklei/go-restful"
```

Comment:
Using `restful "github.com/emicklei/go-restful"` will let us avoid problems with `gulp format`.

Reply:
For me it didn't really make any difference. Sometimes import gets still deleted. Maybe IntelliJ plugin removes aliases and then gulp format removes import.

Reply:
My IntelliJ leaves them prefixed, let's use this format.